scripts/main.ps1

```diff
@@ -173,12 +175,62 @@ Invoke-WPFRunspace -ScriptBlock {
 } | Out-Null
 
 #===========================================================================
-# Setup and Show the Form
+# Print Logo into Console
 #===========================================================================
 
-# Print the logo
 Invoke-WPFFormVariables
 
+#===========================================================================
+# Run Automation
+#===========================================================================
+
+if ($PARAM_CONFIG) {
+    if ($PARAM_RUN) {
+        $installConfig = Get-Content $PARAM_CONFIG -Raw | ConvertFrom-Json
+        if ($installConfig.WPFInstall) {
+            write-host "Installing Programs"
+
+            # Create a new array to hold the combined install configurations
+            $combinedInstallConfig = @()
+
+            # Iterate over each WPFInstall entry
+            for ($i = 0; $i -lt $installConfig.WPFInstall.Count; $i++) {
+                $wpfInstallEntry = $installConfig.WPFInstall[$i]
+                $installEntry = $installConfig.Install[$i]
+
+                # Create a new object with the combined values
+                $combinedEntry = @{
+                    Name = $wpfInstallEntry
+                    Winget = $installEntry.winget
+                    Choco = $installEntry.choco
+                }
+
+                # Add the combined entry to the array
+                $combinedInstallConfig += $combinedEntry
+            }
```

Comment:
Hi! 😊 Great work as always! I suggest using a foreach loop with the range operator for combining the installation configs to make the code shorter
```suggestion
            $combinedInstallConfig = foreach ($i in 0..($installConfig.WPFInstall.Count - 1)) {
                @{
                    Name  = $installConfig.WPFInstall[$i]
                    Winget = $installConfig.Install[$i].winget
                    Choco  = $installConfig.Install[$i].choco
                }
            }
```

Reply:
oh thanks, way better this way :)

Reply:
You're welcome! I'm glad you liked it!